src/applications/proxy-rewrite/proxy-rewrite-whitelist.json

```diff
@@ -74,6 +74,11 @@
         "hostname": "www.cem.va.gov",
         "pathnameBeginning": "/",
         "cookieOnly": false
+    },
+    {
+        "hostname": "www.publichealth.va.gov/",
+        "pathnameBeginning": "/",
+        "cookieOnly": true
```

Comment:
https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/architecture/teamsite/